templates/complete_saml_body.html

```diff
@@ -0,0 +1,23 @@
+{{define "complete_saml_body"}}
+
+<html>
+	<head>
+		<script>
+			document.addEventListener("DOMContentLoaded", function(event) {
+
+                // TODO: targetOrigin should accept specific domains not *
+                // This template will be used for embedded use cases,
+                // For Chrome extension: check complete_saml_extension_body
+				window.opener.postMessage({
+					value: {{.Token}},
+					error: {{.Error}}
+				}, "*");
```

Comment:
this is definitely a stopper!!! if you are going to use `postMessage` at least you need to define the allowed origin, having it like this means that any origin can get the Token, which you shouldn't be doing anyway

Reply:
This was left for further development.  It will not be utilized until we are able to provide a target origin which would require a config property for that to work.  I agree this would be a stopper but I would like to make clear that this is not being utilized.   It's a placeholder for when we start to support other embedded use cases.

You can take a look at the chrome api utlized below, that's the method we are utilizing for sending the token securely